ansible/roles/test/files/saitests/pfc_asym.py

```diff
@@ -143,14 +142,12 @@ def runTest(self):
         # Send packets for lossless priorities from all server ports (src) to non-server port (dst)
         self.sendData(self.server_ports, self.non_server_port, self.router_mac, self.lossless_priorities)
 
-        # Verify that some packets are dropped on dst port, which means that Tx buffer is full
-        port_counters, queue_counters = sai_thrift_read_port_counters(self.client, port_list[int(self.non_server_port['index'])])
-        assert(port_counters[self.EGRESS_DROP] > 0)
-
         # 1. Verify that some packets are dropped on src ports, which means that Rx queue is full
         # 2. Verify that PFC frames are generated for lossless priorities
         for sp in self.server_ports:
-            port_counters, queue_counters = sai_thrift_read_port_counters(self.client, port_list[int(sp['index'])])
+            port_counters, queue_counters = sai_thrift_read_port_counters(self.client, sp['oid'])
+            self.log("\nLossless: port_counters = {}\n queue_counters = {}\nserver port = {}".format(port_counters,
```

Comment:
It will be hard to read, line will be to long.
It is good readable even when there are many ports, because it goes one by one.

Also fixed formatting here.